foreman.te

```diff
@@ -293,10 +299,20 @@ optional_policy(`
 
 tunable_policy(`passenger_can_connect_openstack',`
     ifdef(`distro_rhel6', `
+        # keystone (identity service)
         corenet_tcp_connect_commplex_port(passenger_t)
+        # nova (compute service)
+        # via foreman_allowed_port_t
```

Comment:
The only problem with this is that the boolean won't have an effect on EL6, while it works for EL7 and all other services.  It should probably be separate?